apps/router/src/guardian-ui/admin/FederationAdmin.tsx

```diff
@@ -82,49 +76,14 @@ export const FederationAdmin: React.FC = () => {
             <Skeleton height='32px' width='180px' />
           )}
         </Heading>
-        <Flex
-          flexDirection={{ base: 'column', md: 'row' }}
-          justifyContent='space-between'
-          alignItems={{ base: 'stretch', md: 'center' }}
-          gap={{ base: 4, md: 6 }}
-        >
-          <Box width={{ base: '100%', md: 'auto' }}>
-            <InviteCode inviteCode={inviteCode} />
-          </Box>
-          {config && (
-            <Box width={{ base: '100%', md: 'auto' }}>
-              <BftInfo
-                numPeers={Object.keys(config.global.api_endpoints).length}
-              />
-            </Box>
-          )}
-        </Flex>
-        <Flex
-          gap={6}
-          alignItems='flex-start'
-          flexDir={{ base: 'column', sm: 'column', md: 'row' }}
-        >
-          <Flex w='100%' direction='column' gap={5}>
-            <FederationInfoCard
-              status={status}
-              config={config}
-              latestSession={latestSession}
-            />
-            <BitcoinNodeCard modulesConfigs={config?.modules} />
-          </Flex>
-          <BalanceCard />
-        </Flex>
-        <GuardiansCard
-          status={status}
-          config={config}
-          signedApiAnnouncements={signedApiAnnouncements}
-        />
-        <GatewaysCard config={config} />
+        <InviteCode inviteCode={inviteCode} />
```

Comment:
I was expecting all files to be reformatted to use tabs instead of spaces. :D

Reply:
I can do that separately, that's a linter update